core/arch/arm/kernel/boot.c

```diff
@@ -1482,6 +1509,7 @@ void __weak boot_init_primary_late(unsigned long fdt,
 	} else {
 		init_tee_runtime();
 	}
+	check_crc32_feature();
```

Comment:
I think `check_crc32_feature()` should rather be called from a `boot_final()` registered handler.

Reply:
As discussed above, I removed this commit from the PR.